interface/FrankWriter.h

```diff
@@ -22,10 +22,12 @@ template <class EventClass> class FrankWriter : public BaseOperator<EventClass>
     std::vector<mut::Jet> mix_jets_ = {};
     std::vector<mut::Jet> mix_jets_by_pt_ = {};
     std::vector<mut::Candidate> mix_dijets_ = {};
+    std::vector<double> best_match_distances_ = {};
```

Comment:
with this you can only save the the distance of each hemisphere but not the n best